1223-curator_worklist/1375-curator_views_doi_request_tab_part_ii.feature

```diff
@@ -0,0 +1,12 @@
+    @1375
+    Scenario: Curator views DOI Request tab (Part II)
+        Given that the user opens My Worklist
+        When they click the tab:
+            | DOI request |
+        Then they see a list with expandable items with fields:
+            | DOI Request Status |
+            | Publication Title  |
+            | Messages           |
```

Comment:
What are the included information for each message? 
1) Should we include the date? 
2) Is it necessary to include the name of the sender or is just the role good enough? 
3) In other words should a Creator know who is the Curator or is it irrelevant? 
4) In the picture it is shown that one message (probably the latest) is shown before the item is expanded. Is this necessary?

Reply:
I think there should be one more scenario which describes the messages more thoroughly so that these questions are answered. @terjehell

Reply:
@1252 is updated. Can you see if this answers your questions, @axthosarouris ?